frontend/src/scenes/dashboard/DashboardItem.js

```diff
@@ -244,14 +244,12 @@ export function DashboardItem({
                 </div>
                 <div className="dashboard-item-content">
                     {Element ? (
-                        <div className="graph-container">
```

Comment:
I think the cypress test might be failing because of this

Reply:
That's odd, because this div is just an implementation detail and there doesn't seem to be any reason why it could prevent the success toast from showing up. Actually the success toast does show up for me.

Reply:
Hm yep looks like the test fixed itself 👍